compiler/util/files.cpp

```diff
@@ -347,11 +347,18 @@ bool isChplSource(const char* filename) {
   return retval;
 }
 
+bool isDynoLib(const char* filename) {
+  bool retval = checkSuffix(filename, "dyno");
```

Comment:
Perhaps, but I didn't want to get into a naming discussion for this effort. And this isn't really a user-facing feature since it's more of a first-step in what we need to build towards.